### DISCUSSION TITLE
Local LLMs

### DISCUSSION THREAD

**Discussion by lilhoser** [microsoft/semantic-kernel#173] [General]
When will SK support offline/local LLMs instead of remote ones we must go through OpenAI to access?

**Comment by timlaverty** [comment]
Hi @lilhoser , multi-model support is in PR right now - you should see it in < a week.  @dluc, @dmytrostruk

**Comment by alexchaomander** [comment]
@lilhoser this got merged in with #161

**Reply by lilhoser** [reply]
Thanks! When will it get published to nuget?

**Reply by alexchaomander** [reply]
This should be in the latest one: #r "nuget: Microsoft.SemanticKernel, 0.9.61.1-preview"
